### DISCUSSION TITLE
转帖gitee的tiny-engine上网友提的一个问题【第一次打开8080端口，一直处于“应用数据初始化中。。。】

### DISCUSSION THREAD

**Discussion by kerneltravel** [opentiny/tiny-engine#163] [General]
[第一次打开8080端口，一直处于“应用数据初始化中](https://gitee.com/opentiny/tiny-engine/issues/I8LO3R) 
这个问题我也遇到了，在本地运行的时候。按照官网文档起的，但始终出现websocket 的 ws://127.0.0.1:8080 通讯等待的问题。所以页面一直卡在那里没显示最终设计器内容。 大家有遇到吗？你们是怎么解决的？

**Comment by chilingling** [comment]
方便给一下控制台输出信息吗？

**Reply by kerneltravel** [reply]
![image](https://github.com/opentiny/tiny-engine/assets/437879/e9dbd572-c70d-4f1e-ac74-d033b544fa64)
这个控制台输出信息来自 https://gitee.com/opentiny/tiny-engine/issues/I8LO3R

**Reply by chilingling** [reply]
有终端相关的报错信息吗？或者有没有可以稳定复现的步骤？
@kerneltravel

**Reply by kerneltravel** [reply]
@chilingling 刚我再重试了一下，用当时同样的代码同样的命令运行的，无法复现。
